Py/QuantitativeReporting.py

```diff
@@ -981,18 +981,20 @@ def createJSONFromAnatomicContext(self, terminologyEntry):
     segmentData = dict()
 
     regionObject = terminologyEntry.GetAnatomicRegionObject()
-    if regionObject is None:
+    if regionObject is None or not self.areTerminologyInformationValid(regionObject):
       return {}
     segmentData["AnatomicRegionSequence"] = self.getJSONFromVtkSlicerTerminology(regionObject)
 
     regionModifierObject = terminologyEntry.GetAnatomicRegionModifierObject()
-    if regionModifierObject is not None:
+    if regionModifierObject is not None and self.areTerminologyInformationValid(regionModifierObject):
       segmentData["AnatomicRegionModifierSequence"] = self.getJSONFromVtkSlicerTerminology(regionModifierObject)
-
     return segmentData
 
-  def getJSONFromVtkSlicerTerminology(self, codeSequence):
-    return self.createCodeSequence(codeSequence.GetCodeValue(), codeSequence.GetCodingScheme(), codeSequence.GetCodeMeaning())
+  def areTerminologyInformationValid(self, termTypeObject):
```

Comment:
I would suggest renaming to `terminologyInformationValid` or `isTerminologyInformationValid`